app/models/monitoring/getBusinessDetails.scala

```diff
@@ -22,9 +22,9 @@ import services.monitoring.AuditModel
 object getBusinessDetails {
   val controlListTransactionName = "getBusinessDetails api-5"
 
-  case class getBusinessDetailsModel(nino: String,
-                                     suffix: String,
-                                     responseBody: String
+  case class BusinessDetailsAuditModel(nino: String,
```

Comment:
Better name.  The `get...` name probably came from a method, once upon a time, a long time ago.